### DISCUSSION TITLE
Shortcut to open XML Import page

### DISCUSSION THREAD

**Discussion by mgrak** [arnoudkooi/SN-Utils#432] [Ideas]
/xmlimp
https://{instance}.service-now.com/upload.do

**Comment by arnoudkooi** [comment]
This is one you can easily set as a custom one in the slashcommands tab in the popup, url: 
`/upload.do` to always open in new tab
Or if you use `upload.do` as URL it will try to open in current tab, unless you use CTRL Enter
